pkg/extensions/search/common/oci_layout.go

```diff
@@ -3,52 +3,31 @@ package common
 
 import (
 	"encoding/json"
-	"fmt"
 	"io/ioutil"
 	"os"
 	"path"
 	"strings"
+	"time"
 
 	"github.com/anuvu/zot/errors"
 	"github.com/anuvu/zot/pkg/log"
-	"github.com/anuvu/zot/pkg/storage"
 	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/types"
 	godigest "github.com/opencontainers/go-digest"
 	ispec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
-// CveInfo ...
+// OciLayoutInfo ...
 type OciLayoutUtils struct {
-	Log             log.Logger
-	StoreController storage.StoreController
```

Comment:
I removed storage because this functions can be utilize just by passing blob or index.json path and there should not be any dependency to create a StoreController to use this method.